src/visualization/utils/annotationUtils.ts

```diff
@@ -144,6 +144,18 @@ const makeAnnotationClickHandler = (
   return clickHandler
 }
 
+// want all the ranges *before* all the points,
+// so that can click on points that overlap ranges
+const sortAnnotations = (anno1, anno2) => {
```

Comment:
can you write a test for this? it should be pretty easy to test these cases

Reply:
added